freqtrade/resolvers/iresolver.py

```diff
@@ -91,6 +92,9 @@ def _search_object(cls, directory: Path, *, object_name: str, add_source: bool =
             if not str(entry).endswith('.py'):
                 logger.debug('Ignoring %s', entry)
                 continue
+            if entry.is_symlink() and not Path.joinpath(directory, os.readlink(entry)).exists():
```

Comment:
After being able to recreate this now - I think this could/should be simplified to 
`if entry.is_symlink() and not entry.is_file():`

That would avoid using the os module for file operations (which is kinda fluky on different environments / OS's).
[source](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_file)

Reply:
Yes, that works. I have to admit I did not expect that is_file is True if it is a symlink....